packages/flutter/lib/src/physics/spring_simulation.dart

```diff
@@ -25,6 +28,10 @@ class SpringDescription {
   });
 
   /// Creates a spring given the mass (m), stiffness (k), and damping ratio (ζ).
+  /// The damping ratio describes a gradual reduction in a spring oscillation.
+  /// By using the damping ratio, you can define how rapidly the oscillations
+  /// decay from one bounce to the next.
+  ///
   /// The damping ratio is especially useful trying to determining the type of
```

Comment:
Bonus: I know you didn't write this part, but now would  be a good time to fix the grammar here. "trying to determining" => "when trying to determine"